scss/compounds/_upsell.scss

```diff
@@ -44,21 +45,22 @@
 	}
 
 	&__atc-btn {
-		&:hover {
-			text-decoration: none;
-		}
-
-		&__icon {
+		.sni {
 			font-size: calc-em(30);
 			color: $primary;
+			transition: color $hover-transition-timing;
 		}
 
-		&__text {
+		span {
 			font-size: calc-em(12);
 			margin-top: calc-em(2, 12);
 			text-decoration: underline;
 			color: $body-color;
 		}
```

Comment:
can this code be on 2nd level? for img and __img maybe we can use the same selector?